index.html

```diff
@@ -326,26 +331,35 @@ <h2><dfn>observe()</dfn> method</h2>
         <p>The <a>observe()</a> method instructs the user agent to
         <dfn>register the observer</dfn> and must run these steps:</p>
         <ol data-link-for="PerformanceObserverInit">
-          <li>Let <var>entry types</var> be <var>options</var> <a>entryTypes</a>
-            sequence.</li>
-          <li>Remove all unsupported types from <var>entry types</var>. The user
-            agent SHOULD notify developers if <var>entry types</var> is modified.
-            For example, a console warning listing removed types might be appropriate.</li>
-          <li>If the resulting <var>entry types</var> sequence is an empty sequence,
+          <li>Set <a>observed entry types</a> to a copy of <var>options</var>'
```

Comment:
I think it'd be better to explicitly state what the options variable is a line above that.
e.g. "Let <var>options</var> be the input <a>PerformanceObserverInit</a> object" or something along those lines

Reply:
Also, I think you meant "observed entry types" to be a var, not a link

Reply:
I've added what observer() receives. I really do mean a link to the associated concept of the PerformanceObserver, not a local var. So <a> is correct in this context.

Reply:
*so \<a\> is correct